src/app.ts

```diff
@@ -11,13 +16,55 @@ const SERVER_ADDRESS = '0.0.0.0';
 
 DBConnection()
   .then(async (connection: any) => {
-    // Pre-populate the database with times
+    // Pre-populate the database with times, clubs, interests, and majors
     Constants.VALID_TIMES.forEach(time => MatchingRepo.createTime(time));
+    importDataFromFile('PearClubs.txt', ClubRepo.createClub);
+    importDataFromFile('PearInterests.txt', InterestRepo.createInterest);
+    addCornellMajorsToDB();
+    setupMajorScraperCron();
     app.express.listen(PORT, () => {
       console.log(`App is running on ${SERVER_ADDRESS}:${PORT}...`);
       console.log('Press CTRL-C to stop\n');
     });
   })
   .catch((error: any) => console.log(error));
 
+async function importDataFromFile(
+  filename: string,
+  fn: (data: string) => Promise<void>
+) {
+  const fs = require('fs');
+  const readline = require('readline');
+  const fileStream = fs.createReadStream(
+    __dirname + '/../../assets/' + filename
+  );
+  const rl = readline.createInterface({
+    input: fileStream,
+    crlfDelay: Infinity,
+  });
+  for await (const line of rl) {
+    if (line.trim()) {
+      fn(line);
+    }
+  }
+}
+
+async function addCornellMajorsToDB() {
+  const majors = await scrapeCornellMajors();
+  if (majors.length) {
+    majors.forEach(major => {
+      CornellMajorRepo.createCornellMajor(major);
+    });
+  } else {
+    console.log("Error scraping Cornell's list of majors");
```

Comment:
do you guys not have a `LogUtils.log` where you have more detailed error msg formats? you could take a look at `ithaca-transit-backend`'s LogUtils maybe